fpo-web/src/app/survey/survey.component.ts

```diff
@@ -271,8 +271,22 @@ export class SurveyComponent implements OnInit, OnDestroy {
   }
 
   submitForm(data: any) {
-    //Default to notice to disputant response. Please change later on. 
-    const url = this.dataService.getApiUrl("submit-form/?name=notice-to-disputant-response");
+    // This is horrendous.
```

Comment:
Maybe something like this in SurveyJS (Psuedo code, haven't tried):
    surveyModel.onValueChanged.add((sender, options) => {
    });
survey.setVariable("form", value)
survey.onComplete.add(function(survey, options) {
   survey.setValue("form", survey.getVariable("form"));
});